src/Table/Column.php

```diff
@@ -272,16 +271,15 @@ public function addClass($class, $position = 'body')
     }
 
     /**
-     * Adds a new attribute to the cells of this column. The optional second argument may be "head",
-     * "body" or "foot". If position is not defined, then attribute will be applied on all cells.
```

Comment:
since introduction the comment was wrong - https://github.com/atk4/ui/blob/35052dddd99978efb56d1d9eacb7ed81bc2dcdb9/src/Column/Generic.php#L21